root/static/scripts/release-editor/fields.js

```diff
@@ -961,7 +1008,9 @@ class Release extends mbEntity.Release {
     this.needsTrackInfo = errorField(function () {
       return !self.hasTrackInfo();
     });
-    this.hasInvalidPregapLength = errorField(this.mediums.any('hasInvalidPregapLength'));
+    this.hasInvalidPregapLength = errorField(
+      this.mediums.any('hasInvalidPregapLength'),
+    );
```

Comment:
I would suggest to simply make eslint to ignore the files that need to be replaced because they are based on Knockout and jQuery rather than React.

Reply:
I don't think we want it to fully ignore these - if it did, we also wouldn't find out if someone added a more serious issue that eslint would have caught :) (and we won't get rid of most of them that quickly either).